page/404.njk

```diff
@@ -7,6 +7,8 @@
     <h1 class="govuk-heading-xl">Page not found</h1>
     <p class="govuk-body">If you typed the web address, check it is correct.</p>
     <p class="govuk-body">If you pasted the web address, check you copied the entire address.</p>
-    <p class="govuk-body">If the web address is correct or you selected a link or button, you can contact us for help with your application</p>
+    <p class="govuk-body">If you want to navigate back a page, use the 'back' link on the page</p>
+    <p class="govuk-body">To continue, <a href="\apply">complete your application</a> and check your answers</p>
+    <p class="govuk-body">If the web address is correct or this error persists, you can contact us for help with your application</p>
```

Comment:
Was this written by someone else and we are using it, or did you write this and Julie gave it the thumb-up?

Reply:
I wrote it and Julie approved it.